src/gsy_e/gsy_e_core/simulation.py

```diff
@@ -867,11 +945,11 @@ def _deactivate_areas(self, area: "AreaBase"):
         pass
 
     def _execute_simulation(
-            self, slot_resume: int, tick_resume: int, console: NonBlockingConsole = None) -> None:
-
-        slot_count, slot_resume, tick_resume = (
-            self._time.calculate_total_initial_ticks_slots(
-                self._setup.config, slot_resume, tick_resume, self.area))
+            self, slot_resume: int, _tick_resume: int, console: NonBlockingConsole = None) -> None:
+        gsy_e.constants.RUN_IN_REALTIME = True
```

Comment:
It looks like this line makes every simulation to run in a real time.
```suggestion

```

Reply:
Nice catch! I forgot to remove this after testing. Will do it now. Thanks!!!